.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
   - docker-compose up -d db
   - docker-compose up -d redis
   - sleep 10
-  - docker-compose run -e RAILS_ENV=test web rake db:create
+  - docker-compose run web rake db:create
```

Comment:
Why no longer just 'test'?

Reply:
travis-ci set RAILS_ENV=test and bc the docker-compose now passes RAILS_ENV through, this is redundant.